lib/droplet_kit/resources/kubernetes_resource.rb

```diff
@@ -25,6 +25,7 @@ class KubernetesResource < ResourceKit::Resource
       end
 
       action :cluster_node_pools, 'GET /v2/kubernetes/clusters/:cluster_id/node_pools' do
+        handler(200) { |response| KubernetesNodePoolMapping.extract_collection(response.body, :read) }
```

Comment:
does this need to have `query_keys :per_page, :page, :tag_name`

Reply:
the answer is no it does not. for now. if we do want to add pagination here then we'll need to make it so that the meta tag is returned at these api endpoints. cc @omarismail